securedrop/journalist_templates/admin.html

```diff
@@ -85,7 +85,7 @@ <h2 id="users-heading" class="visually-hidden">Users</h2>
 
 <a href="{{ url_for('admin.manage_config') }}" class="btn icon icon-edit section-spacing-inline"
   id="update-instance-config" aria-label="{{ gettext('Update instance config') }}">
-  {{ gettext('INSTANCE CONFIG') }}
+  {{ gettext('INSTANCE <abbr title="Configuration">CONFIG</abbr>') }}
```

Comment:
Let's go ahead and just expand this, as we have in:  :-)

https://github.com/freedomofpress/securedrop/blob/4d9b74b7e494b6a8e2d06677bc7c20d558b5f078/securedrop/journalist_templates/config.html#L10

```suggestion
  {{ gettext('INSTANCE CONFIGURATION') }}
```

Reply:
fixed this